packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -3,14 +3,17 @@
 @use "./mq";
 @use "./mixins";
 
+/* Import custom fonts */
+@import url("https://fonts.googleapis.com/css2?family=Lato&family=Raleway:wght@400;500;700;900&familydisplay=swap");
```

Comment:
Is this a prototype PR of some kind? I also see commented out code later.

https://github.com/conversionxl/aybolit/blob/master/packages/storybook/.storybook/preview-head.html is font definitions

Reply:
It's definitely a draft PR. The task isn't simply updating typography but implementing global styles. Our components, especially the new ones don't inherit from typography.scss.